src/blog/2023/08/flowfuse-1-11-release.md

```diff
@@ -0,0 +1,80 @@
+---
+title: FlowFuse 1.11 makes it easier to get started with FlowFuse and Node-RED
+subtitle: The new FlowFuse 1.11 release includes a new starter tier for FlowFuse Cloud, Personal Access Tokens for API access and improvements to device management.
+description: The new FlowFuse 1.11 release includes a new starter tier for FlowFuse Cloud, Personal Access Tokens for API access and improvements to device management.
+date: 2023-08-31 
+authors: ["ian-skerrett"]
+image: /blog/2023/08/images/release-1-11-graphic.png
+tags:
+    - posts
+    - flowfuse
+    - releases
+---
+
+FlowFuse 1.11 introduces a new starter tier for FlowFuse Cloud that makes it easier to get started with FlowFuse and Node-RED.
```

Comment:
We introduce the Team tier too?

Reply:
Team tier is just a rename of Premium. Correct? If this is the case, I don't think it is worth mentioning.